### DISCUSSION TITLE
Schema Registry using librdkafka C++ with Protobuf messages

### DISCUSSION THREAD

**Discussion by NikunjGithub** [edenhill/librdkafka#4606] [Q&A]
Hi,
Does librdkafka for C++ provide support for schema registry for Protobuf messages? 
As per online information I could find that libserdes provides support for schema registry for Avro. Is there any solution for Protobuf as well?
https://github.com/confluentinc/libserdes

**Comment by boris-r-v** [comment]
U can try check this one 
https://github.com/boris-r-v/kafka-schema-registry-cpp
